db/seeds.rb

```diff
@@ -206,6 +206,8 @@
 # other available definitions:
 controller_def = Definition.create!(content: "Communicates between the model (database) and views (user interface/website/browser)", word: controller)
 controller_def_2 = Definition.create!(content: "A controller is the waiter; between the model and the view. It determines what the user will be doing/seeing based on the request", word: controller)
+crud=Definition.create!(content:"An acronym for CREATE, READ, UPDATE, and DELETE. These four functions persist and manipulate data in the database", word: crud)
```

Comment:
hey @ChristyTropila , would you be okay with:
a) deleting the "persist and manipulate" phrase? They don't refer to all actions (DELETE does not persist data, READ does not persist or manipulate)
b) changing the word "functions" to "actions"?